src/utils/components/Charts/CPUUtil/CPUThresholdChart.tsx

```diff
@@ -79,17 +71,17 @@ const CPUThresholdChart: FC<CPUThresholdChartProps> = ({ pods, vmi }) => {
     };
   });
 
-  const thresholdData = cpuRequested?.map(([x, y]) => {
-    return {
+  const thresholdData = [
+    {
       name: 'CPU requested',
-      x: new Date(x * MILLISECONDS_MULTIPLIER),
-      y: humanizeCpuCores(Number(y)).value,
-    };
-  });
+      x: new Date(currentTime),
+      y: cpuRequested * 1000,
```

Comment:
Do we need this multiplication? `humanizeCpuCores` did not multiply by 1000 if the value was above 1.

Reply:
If CPU usage is in `m`, we should also write in the chart in `m`

Reply:
I think here we should always show in m the chart . After 999 m, the usage would decome 1 and not 1000. This will lead to some issues in the chart